test_suite/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from indy import did
+
+from modules.connection import Connection
+from modules.basicmessage import BasicMessage
+from modules.trustping import TrustPing
+from agent import Agent
+
+
+@pytest.mark.asyncio
+async def ref_agent(event_loop):
```

Comment:
Having an agent that we can start and use inside the test suite is convenient for testing but I don't think the dependency of the test suite on the reference agent is appropriate, as discussed in PR #104.

Reply:
I mistakenly committed this. Thanks for pointing out.